book/website/collaboration/github/github-readme.md

```diff
@@ -1,12 +1,12 @@
-(rr-github-readme)=
-# README and project communication
+(cl-github-readme)=
+# README and Project Communication
 
 README files are the welcome mat for your project.
-They are the first thing new visitors to your project will see and thus are part of a set of really important documents to make potential contributors feel welcome and invite them to get involved.
+They are the first thing where new visitors to your project will see and, thus are part of a set of really important documents to make potential contributors feel welcome and invite them to get involved.
```

Comment:
```suggestion
They are the first thing new visitors to your project will see and are part of a set of essential documents that make potential contributors feel welcome and invite them to get involved.
```

I ran this sentence through Grammarly and it was flagged as being too wordy. My suggestion is what Grammarly recommends.